pet-nutrition-service/server.js

```diff
@@ -0,0 +1,64 @@
+require('dotenv').config();
+
+const express = require('express');
+const mongoose = require('mongoose');
+const logger = require('pino-http');
+const NutritionFact = require('./nutrition-fact')
+
+main().catch(err => console.log(err));
+
+async function main () {
+  const app = express();
+
+  // Middlware
+  app.use(logger());
+  app.use(express.json());
+
+  // POST: Create a new NutritionFact
+  app.post('/nutrition', async (req, res) => {
```

Comment:
To make things simple, can we just initialize the entries in the mongodb database when the server starts? In this way, we don't need to provide the api.

Reply:
Agree.